templates/beta/package.json

```diff
@@ -16,7 +16,7 @@
     "truffle-hdwallet-provider": "0.0.3"
   },
   "dependencies": {
-    "@aragon/os": "^3.0.3",
+    "@aragon/os": "^3.0.8",
```

Comment:
It's 3.0.6, right? Same for the other ones.

Reply:
we are in 3.0.8 now, see #115